javascript-light-bulb/main.js

```diff
@@ -0,0 +1,14 @@
+const $lightBulb = document.querySelector('.light-on');
+const $background = document.querySelector('.background');
+
+function clickBulb(event) {
+  if ($lightBulb.className === 'light-on') {
+    $lightBulb.className = 'light-off';
+    $background.style.backgroundColor = 'rgb(17, 17, 17)';
```

Comment:
It's fine for this exercise, but remember we want to try to avoid using inline styles because it will always override anything else. I probably would've just created a class with this `background-color` and swap between classes.

Reply:
Got it. Thank you!